symmer/projection/cs_vqe.py

```diff
@@ -306,6 +313,13 @@ def project_onto_subspace(self,
         # instantiate the parent S3_projection class with the stabilizers we are enforcing
         super().__init__(fix_stabilizers)
 
+        if isinstance(insert_rotations, PauliwordOp) and self.unitary_partitioning_method=='LCU':
```

Comment:
Fairly hacky LCU implementaiton for CS-VQE... I think we need to change the structure of CS-VQE slightly. This applies the LCU operator to the input op and sets insert_rotaitons=[]


 What do you think Tim?